livekit_room.proto

```diff
@@ -39,6 +39,9 @@ service RoomService {
 
   // Send data over data channel to participants in a room, Requires `roomAdmin`
   rpc SendData(SendDataRequest) returns (SendDataResponse);
+
+  // Update room metadata, will cause updates to be broadcasted to everyone in the room, Requires `roomAdmin`
+  rpc UpdateRoomMetadata (UpdateRoomMetadataRequest) returns (Room);
```

Comment:
It should returns UpdateRoomMetadataResponse message

Reply:
I think this is actually similar to the other APIs.. where Create/Update/Get returns the object itself without wrapping.

Reply:
ah ok, I thought rather that this API would return nothing